client/html/public/new_ticket.html

```diff
@@ -17,12 +19,13 @@
 
       const body = await fetch('/api/tickets', {
         method: 'POST',
-        headers: {'content-type': 'application/json'},
+        headers: {'content-type': 'application/json', Authorization: makeAuthHeader()},
         body: JSON.stringify(ticket)
       }).then(x => x.json());
 
       window.location.href = `/ticket/${body.id}`;
     };
+    window.createTicket = createTicket;
```

Comment:
Cool! With this gone, we can ditch the cookie processing in all the other backends, too. (Outside of this PR, of course.)

Reply:
Does the React frontend support the authorization header thing?

Reply:
Yeah! It should only be doing headers these days.

Reply:
It happened here, a bit underhanded 😅 https://github.com/StyraInc/styra-demo-tickethub/pull/121/commits/7493f54bdc088fef2d9bbacdc88fe64467c22ebd

Reply:
It does not work for me locally...

```
proxy-react-1        | {"level":"info","ts":1718994886.9129744,"msg":"using config from file","file":"/etc/caddy/Caddyfile"}
proxy-react-1        | {"level":"info","ts":1718994886.9136598,"msg":"adapted config to JSON","adapter":"caddyfile"}
proxy-react-1        | {"level":"warn","ts":1718994886.9137762,"msg":"Caddyfile input is not formatted; run 'caddy fmt --overwrite' to fix inconsistencies","adapter":"caddyfile","file":"/etc/caddy/Caddyfile","line":5}
proxy-react-1        | {"level":"info","ts":1718994886.9144351,"logger":"admin","msg":"admin endpoint started","address":"localhost:2019","enforce_origin":false,"origins":["//localhost:2019","//[::1]:2019","//127.0.0.1:2019"]}
proxy-react-1        | {"level":"info","ts":1718994886.914853,"logger":"http.log","msg":"server running","name":"srv0","protocols":["h1","h2","h3"]}
proxy-react-1        | {"level":"info","ts":1718994886.9155214,"logger":"tls.cache.maintenance","msg":"started background certificate maintenance","cache":"0x4000503c00"}
proxy-react-1        | {"level":"info","ts":1718994886.9158401,"msg":"autosaved config (load with --resume flag)","file":"/config/caddy/autosave.json"}
proxy-react-1        | {"level":"info","ts":1718994886.9159465,"msg":"serving initial configuration"}
proxy-react-1        | {"level":"info","ts":1718994886.918119,"logger":"tls","msg":"cleaning storage unit","storage":"FileStorage:/data/caddy"}
proxy-react-1        | {"level":"info","ts":1718994886.9182148,"logger":"tls","msg":"finished cleaning storage units"}
client-react-1       |
client-react-1       | > styra-tickethub-react@0.3.0 start
client-react-1       | > react-scripts start
client-react-1       |
client-react-1       | Invalid options object. Dev Server has been initialized using an options object that does not match the API schema.
client-react-1       |  - options.allowedHosts[0] should be a non-empty string.
client-react-1 exited with code 1
```

I don't think I've changed anything about it relative to whats in `main`.

Reply:
No local changes? That's weird because it works on main, it's run for every PR: https://github.com/StyraInc/styra-demo-tickethub/actions/runs/9613233537/job/26515559679#step:5:194

Reply:
Having just checkout out `main` and pulled it, I get the same failure with this command ` docker compose --profile csharp --profile react up --build --force-recreate --renew-anon-volumes  2>&1 | tee ~/f/tmp/dcup.log`.

[dcup.log](https://github.com/user-attachments/files/15932310/dcup.log)

Reply:
```
$ git status
On branch main
Your branch is up to date with 'origin/main'.

Untracked files:
  (use "git add <file>..." to include in what will be committed)
        tests/api/report.xml

nothing added to commit but untracked files present (use "git add" to track)
$ git log --format=oneline | head -n 5
7d8324bd0f00b6c7798a8dbe0f69c61589bfb144 build(deps-dev): bump nodemon from 3.1.2 to 3.1.4 in /server/node
b27e66d2661f8bc0009c42fb3554efc6c8e88366 build(deps-dev): bump nodemon from 3.1.2 to 3.1.4 in /client/html
7aac3563e85beb3c04c74af07210afda74c13b61 build(deps-dev): bump @types/node in /tests/e2e in the all group
a3809291dc9e4de4797b62e44469c7c556ed85ab build(deps): bump @styra/opa from 1.0.4 to 1.1.0 in /server/node
9c2434340d5454ef1819f73c4a762a49998842ec build(deps): bump @prisma/client from 5.15.0 to 5.15.1 in /server/node
```

Reply:
We'll get rid of the dev server in our test setup... I've got s branch somewhere, I'll revive it on Monday.

Reply:
#195  ✔️